client/scss/components/_header.scss

```
@@ -15,19 +20,26 @@ header {
   }
 
   h1 {
-    display: inline-flex;
-    flex-wrap: wrap;
-    align-items: center;
-
-    &.icon:before {
-      width: 1em;
-      display: none;
-      margin-inline-end: 0.4em;
-      font-size: 1.5em;
+    position: relative;
+    font-size: 30px;
+    font-weight: 800;
+    line-height: 130%;
```

So we get the correct combination of font size weight & leading, and font size in rem.

```suggestion
		@apply w-h1;
    position: relative;
```